src/SmearedJetProducer.cc

```diff
@@ -126,7 +126,8 @@ double SmearedJetProducer::getSmearFactor(TLorentzVector jet, bigTree & theBigTr
   TLorentzVector genJet = SmearedJetProducer::matchedGenJet(jet, jet.Pt() * jet_resolution, theBigTree);
 
   // Set the smearing factor according to "hybrid" method described in the twiki
-  if (genJet.E() > 0)
+  // https://twiki.cern.ch/twiki/bin/view/CMS/JetResolution
+  if (genJet.Pt() > 10) // nanoAOD applies this cut
```

Comment:
would it be easy to have this requirement as an argument to the skimmer instead? Since we don't know yet if we'll apply this change that could be (arguably) more convenient

Reply:
I've added a ```bool``` to differentiate between both options, setting ```genJet.Pt() > 10``` as default to ensure synchronization with the Pisa framework.